zipline/data/us_equity_loader.py

```diff
@@ -176,18 +190,22 @@ def _get_adjustments_in_range(self, asset, dts, field):
                 ratio = s[1]
             if start < dt <= end:
                 end_loc = dts.searchsorted(dt)
+                adj_loc = end_loc
```

Comment:
out of scope here, but the work done in lines 187-190 should probably be moved into this `if` block, to avoid doing unnecessary work.